test/battle/move_animations/smack_down.c

```diff
@@ -0,0 +1,25 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Move Animation Test: Smack Down works when used 15 times in a row")
+{
+    gBattleTestRunnerState->forceMoveAnim = TRUE;
```

Comment:
I think it would probably be good to add a macro for this. `FORCE_MOVE_ANIM` maybe?